piwik-sdk/src/main/AndroidManifest.xml

```diff
@@ -4,4 +4,14 @@
 
     <uses-permission android:name="android.permission.INTERNET"/>
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+
+    <application>
+        <receiver
+            android:name=".InstallReferrerReceiver"
```

Comment:
nvm, I thought that devs have to declare it explicitly in theirs apps `AndroidManifest.xml`